app.js

```diff
@@ -24,10 +29,16 @@ app.use((err, req, res, next) => {
 
 app.use((err, req, res, next) => {
     if(err.hasOwnProperty('status') && err.hasOwnProperty('msg')) {
+        console.log("caught as a custom error")
```

Comment:
remeber to remove clg's at the end!

Reply:
Wanted to leave this in for future debugging, can I just comment them out next PR?